<!DOCTYPE html><html lang="en-us" >


<head>
  <meta charset="utf-8" />
  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    
    <meta name="generator" content="Wowchemy 5.4.0 for Hugo" />

  
  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
  
      
      <link rel="preload" as="style" href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Mono&family=Roboto:wght@400;700&display=swap">
      <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Mono&family=Roboto:wght@400;700&display=swap" media="print" onload="this.media='all'">
    
  

  <meta name="author" content="The Ballester Group" />
  
  <meta name="description" content="The Ballester Group" />
  <link rel="alternate" hreflang="en-us" href="https://BallesterGroup.github.io/" />

    <meta name="theme-color" content="#1565c0" />
  
  <link rel="stylesheet" href="/css/vendor-bundle.min.f1ecf783c14edc00c9320c205831ad8e.css" media="print" onload="this.media='all'">

      <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha512-W0xM4mr6dEP9nREo7Z9z+9X70wytKvMGeDsj7ps2+xg5QPrEBXC8tAW1IFnzjR6eoJ90JmCnFzerQJTLzIEHjA==" crossorigin="anonymous" media="print" onload="this.media='all'">
    
          <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/highlightjs/cdn-release@10.2.1/build/styles/github.min.css" crossorigin="anonymous" title="hl-light" media="print" onload="this.media='all'">
          <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/highlightjs/cdn-release@10.2.1/build/styles/dracula.min.css" crossorigin="anonymous" title="hl-dark" media="print" onload="this.media='all'" disabled>
        
  
  <link rel="stylesheet" href="/css/wowchemy.246129d782c938a644fe2d653d8a976f.css" />
    <script src="https://identity.netlify.com/v1/netlify-identity-widget.js"></script>
    <link rel="alternate" href="/index.xml" type="application/rss+xml" title="The Ballester Group" />
    <link rel="manifest" href="/manifest.webmanifest" />
  <link rel="icon" type="image/png" href="/media/icon_hu6c6bd35698fee7d0733691beaa13467e_2774_32x32_fill_lanczos_center_3.png" />
  <link rel="apple-touch-icon" type="image/png" href="/media/icon_hu6c6bd35698fee7d0733691beaa13467e_2774_180x180_fill_lanczos_center_3.png" />

  <link rel="canonical" href="https://BallesterGroup.github.io/" />
  <meta property="twitter:card" content="summary" />
  
  <meta property="og:site_name" content="The Ballester Group" />
  <meta property="og:url" content="https://BallesterGroup.github.io/" />
  <meta property="og:title" content="The Ballester Group" />
  <meta property="og:description" content="Ballester Group" /><meta property="og:image" content="https://BallesterGroup.github.io/media/icon_hu6c6bd35698fee7d0733691beaa13467e_2774_512x512_fill_lanczos_center_3.png" />
    <meta property="twitter:image" content="https://BallesterGroup.github.io/media/icon_hu6c6bd35698fee7d0733691beaa13467e_2774_512x512_fill_lanczos_center_3.png" /><meta property="og:locale" content="en-us" />
  
    
      <meta property="og:updated_time" content="2023-01-30T00:00:00&#43;00:00" />
    
  

  

<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "WebSite",
  "potentialAction": {
    "@type": "SearchAction",
    "target": "https://BallesterGroup.github.io?q={search_term_string}",
    "query-input": "required name=search_term_string"
  },
  "url": "https://BallesterGroup.github.io"
}
</script>


  <title>The Ballester Group</title>
</head>


<body id="top" data-spy="scroll" data-offset="70" data-target="#navbar-main" class="page-wrapper   "  >

  
  <script src="/js/wowchemy-init.min.8f76bdc9e086322ed5147724ebba3d06.js"></script>

<aside class="search-modal" id="search">
  <div class="container">
    <section class="search-header">

      <div class="row no-gutters justify-content-between mb-3">
        <div class="col-6">
          <h1>Search</h1>
        </div>
        <div class="col-6 col-search-close">
          <a class="js-search" href="#" aria-label="Close"><i class="fas fa-times-circle text-muted" aria-hidden="true"></i></a>
        </div>
      </div>

      <div id="search-box">
        
        <input name="q" id="search-query" placeholder="Search..." autocapitalize="off"
        autocomplete="off" autocorrect="off" spellcheck="false" type="search" class="form-control"
        aria-label="Search...">
        
      </div>


    </section>
    <section class="section-search-results">

      <div id="search-hits">
        
      </div>

    </section>
  </div>
</aside>



  <div class="page-header">
    


<header class="header--fixed">
  <nav class="navbar navbar-expand-lg navbar-light compensate-for-scrollbar" id="navbar-main">
    <div class="container-xl">
      <div class="d-none d-lg-inline-flex">
        <a class="navbar-brand" href="/">The Ballester Group</a>
      </div>
      <button type="button" class="navbar-toggler" data-toggle="collapse"
              data-target="#navbar-content" aria-controls="navbar-content" aria-expanded="false" aria-label="Toggle navigation">
      <span><i class="fas fa-bars"></i></span>
      </button>
      <div class="navbar-brand-mobile-wrapper d-inline-flex d-lg-none">
        <a class="navbar-brand" href="/">The Ballester Group</a>
      </div>
      
      <div class="navbar-collapse main-menu-item collapse justify-content-start" id="navbar-content">
        <ul class="navbar-nav d-md-inline-flex">
          
          <li class="nav-item">
            <a class="nav-link " href="/#about" data-target="#about"><span>Home</span></a>
          </li>
          <li class="nav-item">
            <a class="nav-link " href="/#methods" data-target="#methods"><span>Methods</span></a>
          </li>
          <li class="nav-item">
            <a class="nav-link " href="/#applications" data-target="#applications"><span>Applications</span></a>
          </li>
          <li class="nav-item">
            <a class="nav-link " href="/#pjb" data-target="#pjb"><span>Dr Ballester</span></a>
          </li>
          <li class="nav-item">
            <a class="nav-link " href="/#people" data-target="#people"><span>Members</span></a>
          </li>
          <li class="nav-item">
            <a class="nav-link " href="/#teaching" data-target="#teaching"><span>Teaching</span></a>
          </li>
          <li class="nav-item">
            <a class="nav-link " href="/#news" data-target="#news"><span>News</span></a>
          </li>
          <li class="nav-item">
            <a class="nav-link " href="/#join" data-target="#join"><span>Join us</span></a>
          </li>
          <li class="nav-item">
            <a class="nav-link " href="/#contact" data-target="#contact"><span>Contact</span></a>
          </li>
        </ul>
      </div>

      <ul class="nav-icons navbar-nav flex-row ml-auto d-flex pl-md-2">

            
        <li class="nav-item">
          <a class="nav-link js-search" href="#" aria-label="Search"><i class="fas fa-search" aria-hidden="true"></i></a>
        </li>
        

        
        
        <li class="nav-item dropdown theme-dropdown">
          <a href="#" class="nav-link" data-toggle="dropdown" aria-haspopup="true" aria-label="Display preferences">
            <i class="fas fa-moon" aria-hidden="true"></i>
          </a>
          <div class="dropdown-menu">
            <a href="#" class="dropdown-item js-set-theme-light">
              <span>Light</span>
            </a>
            <a href="#" class="dropdown-item js-set-theme-dark">
              <span>Dark</span>
            </a>
            <a href="#" class="dropdown-item js-set-theme-auto">
              <span>Automatic</span>
            </a>
          </div>
        </li>
        

        
        

      </ul>

    </div>
  </nav>
</header>


  </div>

  <div class="page-body">
    

<span class="js-widget-page d-none"></span>



  
  <section id="about" class="home-section wg-about  "  >
   <div class="home-section-bg " >
     
   </div>
    <div class="container">

    

<div class="row">
  <div class="col-12 col-lg-4">
    <div id="profile">
      <img class="avatar avatar-circle"
           width="270" height="270"
           src="members/imperial.jpg" alt="The Ballester Group">
      <div class="portrait-title">
        <h2>The Ballester Group</h2>
        <h3>
          <a href="https://www.imperial.ac.uk/bioengineering/" target="_blank" rel="noopener">
          Department of Bioengineering</a></h3>    
        <h3>
          <a href="https://www.imperial.ac.uk/" target="_blank" rel="noopener">
          <span>Imperial College London</span>
          </a>
        </h3>
      </div>
    </div>
  </div>
  <div class="col-12 col-lg-8">
    <h1>Welcome to our website</h1>
    <div class="article-style">
        <p style="text-align:justify;">A major roadblock in early <b>drug design</b> is the need for discovering novel and potent small-molecule drug leads. Many projects do not achieve even one sub-micromolar lead and, in those that do, a single lead is typically obtained with nanomolar potency after several years of expert-led iterative optimisation work (thus, requiring extensive human and financial resources). Therapeutic targets are molecular, e.g. a disease-linked protein, or non-molecular (phenotypic), e.g. a cancer cell line or bacterial culture. For every target, the more drug leads are discovered, the greater the likelihood that at least one will not have any preclinical issue and thus will eventually reach clinical trials.
      </p><p style="text-align:justify;">Drugs overcoming clinical trials still face additional challenges after approval. In cancer, tumour heterogeneity and other inter-patient differences result in large response variability across patients despite having the same cancer type and being administered the same drug treatment. <b>Precision oncology</b> tackles this problem by attempting to identify drug-resistant patients early to consider alternative therapies while sparing them unnecessary side effects. Unfortunately, most cancer drugs do not have response markers and most existing markers are far from being sufficiently predictive.
      </p><p style="text-align:justify;">Our research aims at <b>developing and applying computational methods to boost biomedical discoveries</b> in these areas. We leverage three timely opportunities: artificial intelligence innovation, big data availability, and mature domain knowledge. This website introduces ourselves and the research we carry out in these fascinating areas.    
    </div>

  </div>
</div>

</div>
</section>

  
<section id="methods" class="home-section wg-featured  "  >
  <div class="home-section-bg " >
    </div>
     <div class="container">
       <div class="row  ">
       
         
           <div class="section-heading col-12 col-lg-4 mb-3 mb-lg-0 d-flex flex-column align-items-center align-items-lg-start">
             <h1 class="mb-0">Methods</h1>
             
           </div>
         
 <div class="col-12 col-lg-8">
 

   <div class="media stream-item view-compact">
     <div class="media-body">
       <div class="section-subheading article-title mb-0 mt-0">
         <a>Identifying novel molecules with potent activity on cancer cell lines</a>
       </div>
         <div class="article-style">
          <p style="text-align:justify;font-size: 1.25em">In 2013, we showed <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0061318" target="_blank" rel="noopener">how molecule-cell line datasets can be modelled 
            with machine learning</a>.   
            We also demonstrated that <a href="https://doi.org/10.3389/fchem.2019.00509" target="_blank" rel="noopener">machine learning can predict drug synergy on cancer cell lines as long 
            as data inconsistencies between screening centres are taken into account</a>. We also found evidence 
            suggesting that <a href="https://proceedings.mlr.press/v179/hernandez-hernandez22a.html" target="_blank" rel="noopener">estimating the uncertainty of individual predictions will be an effective way to 
            enhance virtual screening on cancer cell lines</a>. Ongoing research also reveals that graph neural 
            networks outperform other algorithms.</p>
            * Menden, M. P.; Iorio, F.; Garnett, M.; McDermott, U.; Benes, C. H.; Ballester, P. J.; Saez-Rodriguez, J. PLoS One 2013, 8 (4), e61318.<br> 
            * Sidorov, P.; Naulaerts, S.; Ariey-Bonnet, J.; Pasquier, E.; Ballester, P. J. Frontiers in Chemistry 2019, 7, 509.<br> 
            * Hernández-Hernández, S.; Vishwakarma, S.; Ballester, P. J. Proceedings of Machine Learning Research 2022, 179, 1–17.<br><p></p>
        </div>
     </div>
     <div class="ml-3">
      <p></p><p></p>
       <a>
         <img src="/experience/mole.png" height="600" width="250" loading="lazy">
       </a>
     </div>
   </div>

  <div class="media stream-item view-compact">
    <div class="media-body">
      <div class="section-subheading article-title mb-0 mt-0">
        <a>Fighting antimicrobial resistance</a>
      </div>
        <div class="article-style">
          <p style="text-align:justify;font-size: 1.25em">Discovering novel antibiotics for human pathogens 
          is a priority to face the challenge posed by 
          the growing antimicrobial resistance trend. We thus investigate chemistry-informed AI models to 
          screen ultralarge libraries for novel antibiotics against major human pathogens.</p>
        </div>
    </div>
    <div class="ml-3">
      <p></p><p></p>
      <a>
        <img src="/experience/anti.png" height="150" width="200" loading="lazy">
      </a>
    </div>
  </div>

  <div class="media stream-item view-compact">
    <div class="media-body">
      <div class="section-subheading article-title mb-0 mt-0">
        <a>Molecular target prediction</a>
      </div>
        <div class="article-style">   
          <p style="text-align:justify;font-size: 1.25em">This is important to reveal the targets of phenotypic hits or better understand the polypharmacology
          and mechanism of action of drugs with already some known targets. We proposed a <a href="https://doi.org/10.3389/fchem.2016.00015" target="_blank" rel="noopener">benchmark specific
          to target prediction and explain why the widespread use of virtual screening benchmarks is inadequate</a>.
          These issues arise from not having the same objective: target prediction aims at finding the targets bound
          by a molecule, while virtual screening aims at finding the molecules binding to a target. We have also
          shown that <a href="https://www.nature.com/articles/s41598-017-04264-w" target="_blank" rel="noopener">reliability prediction boosts performance</a> and provided a <a href="https://onlinelibrary.wiley.com/doi/abs/10.1111/cbdd.13516" target="_blank" rel="noopener">webserver</a> implementing the method,
          which we are currently updating.</p>
          * Peón, A.; Dang, C. C.; Ballester, P. J. Frontiers in Chemistry 2016, 4, 15.<br> 
          * Peón, A.; Naulaerts, S.; Ballester, P. J. Scientific Reports 2017, 7 (1), 3820.<br>
          * Peón, A.; Li, H.; Ghislat, G.; Leung, K. S.; Wong, M. H.; Lu, G.; Ballester, P. J. Chemical Biology & Drug Design 2019, 94 (1), 1390–1401.<br><p></p>
        </div>
    </div>
    <div class="ml-3">
      <p></p><p></p>
      <a>
        <img src="/experience/mole.png" height="150" width="200" loading="lazy">
      </a>
    </div>
  </div>

  <div class="media stream-item view-compact">
    <div class="media-body">
      <div class="section-subheading article-title mb-0 mt-0">
        <a>Ligand-based identification of novel molecules with target activity</a>
      </div>
        <div class="article-style">
          <p style="text-align:justify;font-size: 1.25em">We developed <a href="https://doi.org/10.1002/jcc.20681" target="_blank" rel="noopener">USR</a>, an ultrafast method to identify molecules with similar 3D shape to a template with activity for the target. 
            We later provided a <a href="https://doi.org/10.1093/nar/gkw320" target="_blank" rel="noopener">webserver</a> to carry out virtual screening using this and related methods.</p>
          * Ballester, P. J.; Richards, W. G. Journal of Computational Chemistry 2007, 28 (10), 1711–1723.<br>
          * Li, H.; Leung, K.-S.; Wong, M.-H.; Ballester, P. J. Nucleic Acids Research 2016, 44 (W1), W436–W441.<br><p></p> 
        </div>
    </div>
    <div class="ml-3">
      <p></p><p></p>
      <a>
        <img src="/experience/ligand.png" height="150" width="200" loading="lazy">
      </a>
    </div>
  </div>
   
  <div class="media stream-item view-compact">
    <div class="media-body">
      <div class="section-subheading article-title mb-0 mt-0">
        <a>Structure-based identification of novel molecules with target activity</a>
      </div>
        <div class="article-style">
          <p style="text-align:justify;font-size: 1.25em">Also known as structure-based virtual screening, it is an 
          effective way to discover
          potential drug leads from their docked poses against the 3D structure of the target. 
          This is typically posed as a binary classification problem and the resulting models 
          are called scoring functions. Virtual screening is hard due to extreme class imbalance 
          and data biases. We showed <a href="https://www.nature.com/articles/srep46710" target="_blank" rel="noopener">how machine learning and data augmentation can improve scoring 
          functions for virtual screening</a>. Strong <a href="https://wires.onlinelibrary.wiley.com/doi/abs/10.1002/wcms.1478" target="_blank" rel="noopener">progress in machine learning for structure-based 
          virtual screening</a> has been made in recent years, also in the <a href="https://doi.org/10.1016/j.cbpa.2021.04.009" target="_blank" rel="noopener">prospective applications</a> of 
          these methods. Scoring functions built with machine learning, instead of the classical 
          ones assuming linearity between features and binding strength, provide <a href="https://doi.org/10.1093/bib/bbaa095" target="_blank" rel="noopener">higher hit rates 
          while identifying molecules with higher potency too</a>. </p>
          * Wójcikowski, M.; Ballester, P. J.; Siedlecki, P. Scientific Reports 2017, 7, 46710.<br> 
          * Li, H.; Sze, K.-H.; Lu, G.; Ballester, P. J. WIREs Computational Molecular Science 2021, e1478.<br> 
          * Ghislat, G.; Rahman, T.; Ballester, P. J. Current Opinion in Chemical Biology 2021, 28–34.<br> 
          * Ain, Q. U.; Aleksandrova, A.; Roessler, F. D.; Ballester, P. J. WIREs Computational Molecular Science 2015, 5 (6), 405–424.<br> 
          * Fresnais, L.; Ballester, P. J. Briefings in Bioinformatics 2020, bbaa095.<br><p></p>  
        </div>
    </div>
    <div class="ml-3">
      <p></p><p></p>
      <a>
        <img src="/experience/complex.png" height="50" width="200" loading="lazy">
        <img src="/experience/complex.png" height="450" width="200" loading="lazy">
        <img src="/experience/complex.png" height="600" width="200" loading="lazy">
      </a>
    </div>
  </div>

  <div class="media stream-item view-compact">
    <div class="media-body">
      <div class="section-subheading article-title mb-0 mt-0">
        <a>Benchmarking virtual screening</a>
      </div>
        <div class="article-style">
          <p style="text-align:justify;font-size: 1.25em">We still cannot reliably estimate the prospective performance of a method from retrospective 
          analysis. We have contributed to this effort by explaining <a href="https://doi.org/10.1016/j.ddtec.2020.09.001" target="_blank" rel="noopener">how to reduce decoy bias</a> or 
          providing a user-friendly protocol to build and benchmark machine-learning scoring 
          functions (In Press). <a href="https://pubs.acs.org/doi/10.1021/acs.jcim.3c00218" target="_blank" rel="noopener">Rigorous in-depth analyses</a> are also important to make progress on this 
          key problem.</p>
          * Ballester, P. J. Drug Discovery Today: Technologies. 2020, 81–87.<br> 
          * Tran-Nguyen, V.-K.; Ballester, P. J. Nature Protocols 2023.<br>
          * Tran-Nguyen, V.-K.; Ballester, P. J. Journal of Chemical Inf. and Model. 2023, 27-28.<br><p></p>   
        </div>
    </div>
    <div class="ml-3">
      <p></p><p></p>
      <a>
        <img src="/experience/vs.jpg" height="150" width="200" loading="lazy">
      </a>
    </div>
  </div>

  <div class="media stream-item view-compact">
    <div class="media-body">
      <div class="section-subheading article-title mb-0 mt-0">
        <a>Binding affinity prediction of protein-ligand complexes from their crystal structures</a>
      </div>
        <div class="article-style">
          <p style="text-align:justify;font-size: 1.25em">Not optimal for virtual screening, as non-binders are practically not 
          considered. However, these 
          structure-based models can be useful for <a href="https://doi.org/10.1002/wcms.1465" target="_blank" rel="noopener">optimising the potency of known small-molecule 
          binders against a target</a>. Having the least confounding factors, this type of data 
          leads to the most robust results. In 2010, we discovered that <a href="https://doi.org/10.1093/bioinformatics/btq112" target="_blank" rel="noopener">machine-learning scoring 
          functions are substantially more predictive than classical scoring functions</a> at this 
          task. This <a href="https://doi.org/10.1093/bioinformatics/btz183" target="_blank" rel="noopener">performance gap grows with larger training sets</a>, which is still 
          true when one removes test complexes with similar molecules or targets to those in 
          the training set. We also found that <a href="https://pubs.acs.org/doi/10.1021/ci500091r" target="_blank" rel="noopener">expert-based selection of features, the 
          innovation engine of classical scoring functions, is actually suboptimal</a> for machine-learning scoring functions.</p>
          * Li, H.; Sze, K.-H.; Lu, G.; Ballester, P. J. WIREs Computational Molecular Science 2020, 10 (5), e1465.<br> 
          * Ballester, P. J.; Mitchell, J. B. O. Bioinformatics 2010, 26 (9), 1169–1175.<br>
          * Li, H.; Peng, J.; Sidorov, P.; Leung, Y.; Leung, K. S.; Wong, M. H.; Lu, G.; Ballester, P. J. Bioinformatics 2019, 35 (20), 3989–3995.<br> 
          * Ballester, P. J.; Schreyer, A.; Blundell, T. L. Journal of Chemical Information and Modeling 2014, 54 (3), 944–955.<br><p></p> 
        </div>
    </div>
    <div class="ml-3">
      <p></p><p></p>
      <a>
        <img src="/experience/complex.png" height="150" width="200" loading="lazy">
      </a>
    </div>
  </div>


  <div class="media stream-item view-compact">
    <div class="media-body">
      <div class="section-subheading article-title mb-0 mt-0">
        <a>Structure-based drug repositioning</a>
      </div>
        <div class="article-style">
          <p style="text-align:justify;font-size: 1.25em"><a href="https://doi.org/10.1111/acel.12626" target="_blank" rel="noopener">Machine-learning scoring functions as a part of holistic efforts to find other uses for existing drugs</a>, for instance as anti-aging treatments.</p>
        * Ziehm, M.; Kaur, S.; Ivanov, D. K.; Ballester, P. J.; Marcus, D.; Partridge, L.; Thornton, J. M. Aging Cell 2017.<br><p></p>  
        </div>
    </div>
    <div class="ml-3">
      <p></p><p></p>
      <a>
        <img src="/experience/rep.png" height="150" width="200" loading="lazy">
      </a>
    </div>
  </div>

  <div class="media stream-item view-compact">
    <div class="media-body">
      <div class="section-subheading article-title mb-0 mt-0">
        <a>Enabling drug-based omics-informed precision oncology</a>
      </div>
        <div class="article-style">
          <p style="text-align:justify;font-size: 1.25em">Cancer drugs are only effective in a fraction of patients. 
          We thus need to predict which patients will be resistant to a given drug. Such prediction is based on the 
          presence of a somatic mutation (mutation marker) or the a priori combined expression 
          of expert-selected genes (gene expression signature), but <a href="https://doi.org/10.1093/bib/bbab450" target="_blank" rel="noopener">considering other genes, 
          profiles and model-building approaches improves it</a>. Especially in terms of 
          <a href="https://www.oncotarget.com/article/20923/text/" target="_blank" rel="noopener">recall</a>. A promising alternative is using <a href="https://www.nature.com/articles/s41698-021-00216-w" target="_blank" rel="noopener">AI to predict how patients respond 
          to a drug</a> from the multiple molecular profiles (multi-omics) of their tumours. 
          This AI approach has so far provided enhanced prediction in <a href="https://doi.org/10.3389/fgene.2019.01041" target="_blank" rel="noopener">paclitaxel-treated 
          breast cancer</a> patients, <a href="https://doi.org/10.1002/advs.202201501" target="_blank" rel="noopener">doxorubicin-treated breast cancer</a> patients, 
          gemcitabine-treated pancreatic cancer patients and temolozomide-treated low-grade-glioma 
          patients. Overall, we found that the best learning algorithm - molecular profile combination 
          changes across drugs and cancer types and is also easy to miss as almost all other 
          combinations are not predictive. This highlights the critical importance of considering 
          many algorithms and profiles when analysing these clinical pharmaco-omic datasets.</p>
          * Ballester, P. J.; Stevens, R.; Haibe-Kains, B.; Huang, R. S.; Aittokallio, T. Briefings in Bioinformatics 2021, bbab450.<br>
          * Naulaerts, S.; Dang, C. C.; Ballester, P. J. Oncotarget 2017, 8 (57), 97025–97040.<br> 
          * Ballester, P. J.; Carmona, J. npj Precision Oncology 2021, 5 (1), 1–3.<br> 
          * Bomane, A.; Gonçalves, A.; Ballester, P. J. Frontiers in Genetics 2019, 10, 1041.<br> 
          * Ogunleye, A. Z.; Piyawajanusorn, C.; Gonçalves, A.; Ghislat, G.; Ballester, P. J. Advanced Science 2022, 9 (24), e2201501.<br><p></p>  
        </div>
    </div>
    <div class="ml-3">
      <p></p><p></p>
      <a>
        <img src="/experience/ono.jpg" height="150" width="200" loading="lazy">
      </a>
    </div>
  </div>
  

  
  
    </div>
    </div>
   </div>
   </section>  
  
  


<section id="applications" class="home-section wg-featured  "  >
  <div class="home-section-bg " >
    </div>
     <div class="container">
       <div class="row  ">
          <div class="section-heading col-12 col-lg-4 mb-3 mb-lg-0 d-flex flex-column align-items-center align-items-lg-start">
          <h1 class="mb-0">Applications</h1>
             
           </div>

 
 
 <div class="col-12 col-lg-16">

  <div class="media stream-item view-compact">
    <div class="media-body">


      <div class="article-style">
        <p><p style="text-align:justify; font-size: 1.25em"> Prospectively applying our developed methods is important as a reality check, 
          understanding how to improve methods further and actually making biomedical 
          discoveries. However, it requires finding a collaborator to carry out in 
          vitro tests and also provide expertise on the target. We are hence keen 
          to enter this type of collaborations and have been able to forge some 
          over the years. These are some examples below (we do not include the 
          applications by others using our methods):</p></p>
    </div>

      <div class="section-subheading article-title mb-0 mt-0">
        <a>Discovering novel cell-active molecules on cancer cell lines</a>
      </div>
        <div class="article-style">
          <p style="text-align:justify;font-size: 1.25em">Screening over a billion make-on-demand compounds with our AI models has led to the identification of nanomolar inhibitors of colorectal and glioma cell lines (all with novel chemical scaffolds). In vitro tests are not yet completed.
          </p></div>
      <div class="section-subheading article-title mb-0 mt-0">
        <a>Discovering new molecular targets of drugs</a>
      </div>
        <div class="article-style">
          <p style="text-align:justify;font-size: 1.25em">The targets of these phenotypic hits will be determined with our target prediction 
            method, which already discovered low-nanomolar targets of drugs such as <a href="https://febs.onlinelibrary.wiley.com/doi/full/10.1002/1878-0261.12810" target="_blank" rel="noopener">mebendazole</a> and <a href="https://www.mdpi.com/2218-273X/10/11/1570" target="_blank" rel="noopener">actarit</a>.
          </p>
          * Ariey‐Bonnet, J.; Carrasco, K.; Le Grand, M.; Hoffer, L.; Betzi, S.; Feracci, M.; Tsvetkov, P.; Devred, F.; Collette, Y.; Morelli, X.; Ballester, P.J.; Pasquier, E. Molecular Oncology 2020, 14 (12), 3083–3099.<br> 
          * Ghislat, G.; Rahman, T.; Ballester, P. J. Biomolecules 2020, 10 (11), 1–11.<br><p></p>  
        </div>
      <div class="section-subheading article-title mb-0 mt-0">
        <a>Discovering novel target-active molecules</a>
      </div>
        <div class="article-style">
          <p style="text-align:justify;font-size: 1.25em">USR-based virtual screening has <a href="https://doi.org/10.1098/rsif.2009.0170" target="_blank" rel="noopener">discovered novel low-micromolar inhibitors for a breast cancer target</a> , 
          a <a href="https://doi.org/10.1016/j.ejmech.2014.08.060" target="_blank" rel="noopener">colorectal metastasis target</a> and <a href="https://doi.org/10.1098/rsif.2012.0569" target="_blank" rel="noopener">antibacterial targets</a>, among others. This ligand-based approach was     
          also successful when combined with a generic machine-learning scoring function on these antibacterial 
          targets. We have also applied prospectively PARP1-specific machine-learning scoring functions, which 
          have led so far to cell-active PARP1 small-molecule inhibitors as potent as 1 nanomolar.</p>
          * Ballester, P. J.; Westwood, I.; Laurieri, N.; Sim, E.; Richards, W. G. Journal of the Royal Society Interface 2010, 7 (43), 335–342.<br>  
          * Hoeger, B.; Diether, M.; Ballester, P. J.; Köhn, M. European Journal of Medicinal Chemistry 2014, 88, 89–100.<br>  
          * Ballester, P. J.; Mangold, M.; Howard, N. I.; Robinson, R. L. M.; Abell, C.; Blumberger, J.; Mitchell, J. B. O. Journal of the Royal Society Interface 2012, 9 (77), 3196–3207.<br><p></p> 
        </div>
        
        <div class="article-style">
          <p style="text-align:justify;font-size: 1.25em">On the other hand, we are keen in collaborating with 
          clinicians in drug-based omics-informed precision oncology. The main obstacle to making an impact 
          in this area is now clinical data sharing. Full data from two cohorts of patients with the same 
          cancer type, with their tumours profiled with the same omics technology and treated with the same 
          drug is hard to access. Collaborations providing such data to get our predictive models closer to 
          the clinic is our priority. We would be also happy to consider other drug-cancer type cases for 
          analysis if non-public data is available. Lastly, our AI protocols can be adapted to the analysis 
          of longitudinal molecularly-profiled liquid-biopsy data for early detection of cancer. we are 
          initiating such collaborations and are keen to consider others in this area.</p>
      </div>
    </div>
  </div>
</div>   



</div>
</div>
</section>    

  

  

  
<section id="pjb" class="home-section wg-about">
  <div class="container">
  <div class="row">
    
    <div class="row">
      <div class="col-12 col-lg-4">
        <div id="profile">
          <img class="avatar avatar-circle"
               src="/members/pjb.jpg" alt="The Ballester Group">
          <div class="portrait-title">
            <h2>Dr Pedro Ballester</h2>
            <h3>Royal Society Wolfson Fellow & Senior Lecturer</h3>
            <h3>
              <a href="https://www.imperial.ac.uk/" target="_blank" rel="noopener">
              <span>Imperial College London</span>
              </a>
            </h3>
          </div>
    
          <ul class="network-icon" aria-hidden="true">
            <li>
              <a href="mailto:p.ballester@imperial.ac.uk"  aria-label="envelope">
                <i class="fas fa-envelope big-icon"></i>
              </a>
            </li>
            
            <li>
              <a href="https://scholar.google.com/citations?user=PnrbTyYAAAAJ&hl=en" target="_blank" rel="noopener" aria-label="graduation-cap">
                <i class="fas fa-graduation-cap big-icon"></i>
              </a>
            </li>

            <li> 
              <a href="https://www.imperial.ac.uk/people/p.ballester" target="_blank" rel="noopener" aria-label="home">
                <i class="fas fa-home big-icon"></i>
              </a>
            </li>

            <li>
              <a href="https://twitter.com/pjballester" target="_blank" rel="noopener" aria-label="twitter">
                <i class="fab fa-twitter big-icon"></i>
              </a>
            </li>
    
            <li>
              <a href="https://www.linkedin.com/in/pedroballester/" target="_blank" rel="noopener" aria-label="linkedin">
                <i class="fab fa-linkedin big-icon"></i>
              </a>
            </li>
          </ul>   
        </div>
      </div>
      <div class="col-12 col-lg-8">
        <div class="article-style">
         <p style="text-align:justify;">Following a first degree in Astrophysics and an MSc in Information Processing and Neural Networks, I completed 
            a PhD at Imperial College on geophysical data mining and inference. After that, I was awarded a 
            Junior Research Fellowship at the University of Oxford and then held a short postdoc at the University of 
            Cambridge, both associated with their Chemistry departments. In 2010, I was awarded a 4-year MRC Methodology 
            Research Fellowship, which funded my staff scientist position at the European Bioinformatics Institute, and 
            also a Governing Body Fellowship at Wolfson College Cambridge. In 2014, I moved to the south of France for a position as a tenured group leader at INSERM with A*MIDEX Excellence Chair and ANR Tremplin-ERC 
            awards. In 2022, I returned to the UK as an Associate Professor at Imperial College London and Wolfson Fellow of the Royal Society.</p>

            
          <h3>Awards</h3>
            <p style="text-align:justify;">• 2022-27 Royal Society Wolfson Fellow, UK.<br>
            •	2019-22 INSERM scientific excellence merit award (PEDR), France.<br>
            •	2017-19 ANR Tremplin - ERC, France.<br>
            •	2015-17 A*MIDEX Excellence Chair, France.<br>
            •	2014 Civil Servant Qualifying Exams (INSERM CR1), France.<br>
            •	2011-14 JRF & Governing Body Fellow, Wolfson College Cambridge, UK.<br>
            •	2010-14 MRC Methodology Research Fellow, UK.<br>
            •	2007-08 Junior Research Fellow (JRF), St Cross College Oxford, UK.<br>
            •	2000-01 Scholar, Sa Nostra Foundation, Spain.<br>
            •	2000 Civil Servant Qualifying Exams (Physics Innovation Expert), Spain.</p>

        <h3>Funding</h3>
            <p style="text-align:justify;">I have been awarded a total of €4,314,000 (€2,552,000 plus £1,554,782) to fund my research since 2010 (this includes the funding awarded to my group only, no total grant value). 
              This is broken down into major project grants awarded as PI:<br>
            •	2023-26 EPSRC Healthcare Technologies (UK, £625,936). <br>
            •	2022-27 Royal Society Wolfson Fellowship (UK, £281,280).<br>
            •	2021-24 Plancancer MIC (France, €276,493).<br>
            •	2020-23 ANR PRC grant (France, €320,000). <br>
            •	2019-22 CEFIPRA (India-France bilateral projects, €77,000).<br>
            •	2017-20 ANR Tremplin-ERC (France, €130,000).<br>
            •	2015-17 A*MIDEX Excellence Chair (France, €235,000).<br>
            •	2010-14 MRC Methodology Research Fellowship (UK, £507,936).<br>
            Project grants awarded as co-I:<br>
            •	2022 ARC – SIGN'IT 2022 (France, €110,000).<br>
            •	2022 ANR – PEPR Digital health (France, €500,000).<br>
            •	2016 Canceropôle PACA (France, €83,000).<br>
            And also awarded grants for postdoctoral fellowships (ARC, FRM), PhD scholarships (Chulabhorn Royal Academy, ANR Artificial Intelligence, IPC, HEC, CONACyT, PTDF, USTH) and other purposes (Pfizer, PHC Merlion, INSERM).
            </p>
        
          <h3>Publications and patents</h3>
            <p>As June 2023: 83 peer-reviewed papers since 2003, 79% as corresponding author.                                                                                                  
            H-index restricted to papers where I am either first, last or corresponding author is 29.</p>                 
            <p>Ballester, P.J. (2012) US Patent 8244483 <a href="http://www.google.com/patents/US8244483" target="_blank" rel="noopener"> 
            “Shape Recognition Methods and Systems for Searching Molecular Databases”  </a>
            </p>

          <h3>Community service </h3>  
            <p style="text-align:justify;"><b>Editorial roles:</b> 2021-date <a href="https://academic.oup.com/bib/pages/Editorial_Board" target="_blank" rel="noopener">Editorial Board Member </a>
              of Briefings in Bioinformatics, 2019-date <a href="https://www.nature.com/npjprecisiononcology/about/editors" target="_blank" rel="noopener">Associate Editor</a> 
              of npj Precision Oncology, among <a href="https://www.webofscience.com/wos/author/record/A-1148-2008" target="_blank" rel="noopener">others</a>.
              <b>Peer-review:</b> 480+ reviews of journal papers <a href="https://www.webofscience.com/wos/author/record/A-1148-2008" target="_blank" rel="noopener">verified.</a>
              <b>Grant proposal evaluations for a range of funding agencies:</b> Canada Research Chair, Medical Research Council (MRC, UK), DBT/Wellcome Trust India Alliance (India), 
              Science Foundation (GACR, Czech Republic), National Research Foundation (NRF, South Africa), Ministry of Economy and Competitiveness (MINECO, Spain), Natural Sciences and Engineering Research Council 
              (NSERC, Canada), Agency for Health Quality and Assessment of Catalonia (AQuAS, Spain), Engineering and Physical Sciences Research Council (EPSRC, UK), National Foundation for Science & Technology 
              (FCT, Portugal), Health Research Board (HRB, Ireland), National Research Fund (FNR, Luxembourg), Israel Science Foundation (ISF), The Swiss National Science Foundation, Netherlands Organisation for 
              Scientific Research, National Agency for Research (ANR, France), National Agency for Research (ANEP, Spain), Biotechnology and Biological Sciences Research Council (BBSRC, UK) and National Scientific 
              Research Council  (Romania). <b>Times Higher Education’s World University Rankings: </b> Invited to act as a reputation evaluator every year since 2018.
            </p>
        </div>
</div></div></section>
  

 <section id="people" class="home-section wg-people">
  <div class="container"><div class="row justify-content-center people-widget">
    <div class="col-md-12 section-heading"><h1>Members</h1></div>
    
    
    <div class="col-md-12">
      <h2 class="mb-4">Principal Investigator</h2></div><div class="col-12 col-sm-auto people-person">
        <a href="https://www.linkedin.com/in/pedroballester/" target="_blank" rel="noopener"><img class="avatar avatar-circle" src="/members/pjb.jpg" alt="pjb"></a>
        <div class="portrait-title"><h2><a href="#pjb">Dr Pedro Ballester</a></h2>
        <h3>Royal Society Wolfson Fellow & Senior Lecturer</h3></div></div>
    

    <div class="col-md-12"><h2 class="mb-4">Postdoctoral researchers</h2></div>

    <div class="col-12 col-sm-auto people-person">
      <a href="" target="_blank" rel="noopener"><img class="avatar avatar-circle" src="members/postdoc.jpg" alt="Avatar"></a><div class="portrait-title">
        <h2><a href="https://www.jobs.ac.uk/job/DAT978/research-associate" target="_blank" rel="noopener">Hiring</a></h2>
        <h3>October 2023-date</h3></div></div>

    <div class="col-12 col-sm-auto people-person">
      <a href="" target="_blank" rel="noopener"><img class="avatar avatar-circle" src="members/viet.jpeg" alt="Avatar"></a><div class="portrait-title">
        <h2><a href="https://www.linkedin.com/in/viet-khoa-tran-nguyen/" target="_blank" rel="noopener">Viet-Khoa Tran-Nguyen</a></h2>
        <h3>October 2021-date</h3></div></div>
    
    <div class="col-12 col-sm-auto people-person">
      <a href="" target="_blank" rel="noopener"><img class="avatar avatar-circle" src="members/jitendra.jpeg" alt="Avatar"></a><div class="portrait-title">
        <h2><a href="https://www.linkedin.com/in/jitendra-kuldeep-9a621954/" target="_blank" rel="noopener">Jitendra Kuldeep</a></h2>
        <h3>December 2021-date </h3></div></div>

    <div class="col-md-12"><h2 class="mb-4">PhD Students</h2></div>

    <div class="col-12 col-sm-auto people-person"><a href="https://www.linkedin.com/in/saiveth-hern%C3%A1ndez-hern%C3%A1ndez-491896183/" target="_blank" rel="noopener"><img class="avatar avatar-circle" src="/members/saiveth.jpg" alt="Avatar"></a>
      <div class="portrait-title"><h2><a href="https://www.linkedin.com/in/saiveth-hern%C3%A1ndez-hern%C3%A1ndez-491896183/" target="_blank" rel="noopener">Saiveth Hernández</a></h2>
        <h3>Oct 2020 - date</h3><ul class="network-icon" aria-hidden="true"></a></li></ul></div></div>
    
    <div class="col-12 col-sm-auto people-person"><a href="https://www.linkedin.com/in/chayanit-piyawajanusorn/" target="_blank" rel="noopener"><img class="avatar avatar-circle" src="/members/chayanitp.jpg" alt="Avatar"></a>
      <div class="portrait-title"><h2><a href="https://www.linkedin.com/in/chayanit-piyawajanusorn/" target="_blank" rel="noopener">Chayanit Piyawajanusorn</a></h2>
        <h3>Oct 2022 - date</h3><ul class="network-icon" aria-hidden="true"></a></li></ul></div></div>
    
    <div class="col-12 col-sm-auto people-person"><a href="https://www.linkedin.com/in/hanqin-du-670b13207/" target="_blank" rel="noopener"><img class="avatar avatar-circle" src="/members/hanqind.jpg" alt="Avatar"></a>
      <div class="portrait-title"><h2><a href="https://www.linkedin.com/in/hanqin-du-670b13207/" target="_blank" rel="noopener">Hanqin Du</a></h2>
        <h3>Oct 2022 - date</h3><ul class="network-icon" aria-hidden="true"></a></li></ul></div></div>
    
    <div class="col-12 col-sm-auto people-person"><a href="https://www.linkedin.com/in/qianrong-guo/" target="_blank" rel="noopener"><img class="avatar avatar-circle" src="/members/qianrongg.jpg" alt="Avatar"></a>
      <div class="portrait-title"><h2><a href="https://www.linkedin.com/in/qianrong-guo/" target="_blank" rel="noopener">Qianrong Guo</a></h2>
        <h3>Jan 2023 - date</h3><ul class="network-icon" aria-hidden="true"></a></li></ul></div></div>
    
    <div class="col-12 col-sm-auto people-person"><a href="https://www.linkedin.com/in/klaudia-caba-896662203/" target="_blank" rel="noopener"><img class="avatar avatar-circle" src="/members/klaudiac.jpg" alt="Avatar"></a>
      <div class="portrait-title"><h2><a href="https://www.linkedin.com/in/klaudia-caba-896662203/" target="_blank" rel="noopener">Klaudia Caba</a></h2>
        <h3>April 2023 - date</h3><ul class="network-icon" aria-hidden="true"></a></li></ul></div></div>
    
    <div class="col-12 col-sm-auto people-person"><a href="https://www.linkedin.com/in/krinos/" target="_blank" rel="noopener"><img class="avatar avatar-circle" src="/members/zeruil.jpg" alt="Avatar"></a>
      <div class="portrait-title"><h2><a href="https://www.linkedin.com/in/krinos/" target="_blank" rel="noopener">Zerui Li</a></h2>
        <h3>April 2023 - date</h3><ul class="network-icon" aria-hidden="true"></a></li></ul></div></div>
    
          
    <div class="col-md-12"><h2 class="mb-4">Master Students</h2></div>
    
    <div class="col-12 col-sm-auto people-person"><a href="https://www.linkedin.com/in/cosmin-gabriel-ichim-0b18601b9/" target="_blank" rel="noopener"><img class="avatar avatar-circle" src="/members/cosmini.jpg" alt="Avatar"></a>
      <div class="portrait-title"><h2><a href="https://www.linkedin.com/in/cosmin-gabriel-ichim-0b18601b9/" target="_blank" rel="noopener">Cosmin Ichim</a></h2>
        <h3>MRes in Bioengineering (June 2023 - date)</h3><ul class="network-icon" aria-hidden="true"></a></li></ul></div></div>
      
    <div class="col-12 col-sm-auto people-person"><a href="http://%20https//www.linkedin.com/in/alec-reygrobellet-9930921b9/" target="_blank" rel="noopener"><img class="avatar avatar-circle" src="/members/alecr.jpg" alt="Avatar"></a>
      <div class="portrait-title"><h2><a href="http://%20https//www.linkedin.com/in/alec-reygrobellet-9930921b9/" target="_blank" rel="noopener">Alec Reygrobellet</a></h2>
        <h3>MSc in Bioengineering (January 2023 - date)</h3><ul class="network-icon" aria-hidden="true"></a></li></ul></div></div>
          
    <div class="col-12 col-sm-auto people-person"><a href="https://www.linkedin.com/in/lucas-he-5b7a22265/" target="_blank" rel="noopener"><img class="avatar avatar-circle" src="/members/lucash.jpg" alt="Avatar"></a>
      <div class="portrait-title"><h2><a href="https://www.linkedin.com/in/lucas-he-5b7a22265/" target="_blank" rel="noopener">Lucas He</a></h2>
        <h3>MSc in Bioengineering (January 2023 - date)</h3><ul class="network-icon" aria-hidden="true"></a></li></ul></div></div>
          
    <div class="col-12 col-sm-auto people-person"><a href="https://www.linkedin.com/in/amrit-jandu-45a076270/" target="_blank" rel="noopener"><img class="avatar avatar-circle" src="/members/amritj.jpg" alt="Avatar"></a>
      <div class="portrait-title"><h2><a href="https://www.linkedin.com/in/amrit-jandu-45a076270/" target="_blank" rel="noopener">Amrit Jandu</a></h2>
        <h3>MSc in Digital Chemistry (April 2023 - date)</h3><ul class="network-icon" aria-hidden="true"></a></li></ul></div></div>
          
    <div class="col-12 col-sm-auto people-person"><a href="http://linkedin.com/in/nicola-chesterman-95ab70207" target="_blank" rel="noopener"><img class="avatar avatar-circle" src="/members/nicolac.jpg" alt="Avatar"></a>
      <div class="portrait-title"><h2><a href="http://linkedin.com/in/nicola-chesterman-95ab70207" target="_blank" rel="noopener">Nicola Chesterman</a></h2>
        <h3>MSc in Digital Chemistry (April 2023 - date)</h3><ul class="network-icon" aria-hidden="true"></a></li></ul></div></div>
    
    <div class="col-12 col-sm-auto people-person"><a href="https://www.linkedin.com/in/rasmus-hildebrandt/" target="_blank" rel="noopener"><img class="avatar avatar-circle" src="/members/rasmush.jpg" alt="Avatar"></a>
      <div class="portrait-title"><h2><a href="https://www.linkedin.com/in/rasmus-hildebrandt/" target="_blank" rel="noopener">Rasmus Hildebrandt</a></h2>
        <h3>MSc in Bioinformatics (April 2023 - date)</h3><ul class="network-icon" aria-hidden="true"></a></li></ul></div></div>

    <div class="col-md-12"><h2 class="mb-4">Alumni</h2></div></div>
    <p style="text-align:justify;"><b>Undergraduate students at Imperial (UK): </b>Jonathan Talbot-Martin (2023, primarily supervised by Dr Nicky Whiffin, University of Oxford), 
    Aparna Loecher (2022, primarily supervised by Dr Nuria Oliva-Jorge, Imperial College).</p>
    <p style="text-align:justify;"><b>Postdocs at CRCM (France): </b>Jitendra Kuldeep (2021, 3 years), Viet-Khoa Tran-Nguyen (2021, 2 years), Ghita Ghislat (2022, 9 months), 
    Junaid Muhammad (2021, 18 months), Saw Simeon (2019, 2 years), Pavel Sidorov (2017, 2 years), Stefan Naulaerts (2017, 1.5 years), 
    Cuong Dang (2015, 2 years), Antonio Peon (2015, 2 years).</p>
    <p style="text-align:justify;"><b>PhD students at CRCM (France):</b> Hongjian Li (visiting from CUHK), Linh Nguyen, Adeolu Ogunleye, Sachin Vishwakarma, Saiveth Hernández.</p>    
    <p style="text-align:justify;"><b>Master students at CRCM (France): </b> Louai Kassa-Baghdouche, Dina Ouahbi, Pablo Gomez, Louison Fresnais, Amad Diouf, Michal Zulcinski, Nicola Jaume. </p> 
    <p style="text-align:justify;"><b>Master students at EMBL-EBI (UK): </b> Raquel Romero, Saumya Kumar.</p>          
    <p style="text-align:justify;"><b>Examples of current alumni positions: </b>lecturers/assistant professors (Linh Nguyen and Cuong Dang at Vietnam National 
      University, Pavel Sidorov at Hokkaido University, Viet-Khoa Tran-Nguyen at Paris Cite University), pharmaceutical companies (Sachin Vishwakarma 
      at Evotec), senior Horizon Europe Marie Curie fellowships (Ghita Ghislat at Imperial College London).
    </p>
    </section>  
  
  
<section id="teaching" class="home-section wg-featured  "  >
  <div class="home-section-bg " >
    </div>
     <div class="container">
       <div class="row  ">
          <div class="section-heading col-12 col-lg-4 mb-3 mb-lg-0 d-flex flex-column align-items-center align-items-lg-start">
          <h1 class="mb-0">Teaching</h1></div>
          <div class="col-12 col-lg-16">
            <div class="portrait-title"><p> </p>
           <p style="text-align:justify;">•	2024-25 module creator, leader and lecturer of <i>Artificial intelligence for drug discovery and therapy</i>.</p>
           <p style="text-align:justify;">•	2023-24 module leader of <i>Mathematics 1</i> (BIOE40004) and <i>Maths and Engineering 1</i> (BIOE40005).</p>
           <p style="text-align:justify;">•	2022-24 module lecturer of <i>Mathematics 1</i> (BIOE40004) and <i>Maths and Engineering 1</i> (BIOE40005).</p>
           <p style="text-align:justify;">•	2022-24 guest lecturer of <i>Nanotechnologies for cancer diagnosis and cancer therapy</i> (BIOE97173).</p>
           <p style="text-align:justify;">•	2016 Introductory talk to first-year master students engineering students from École Centrale de Marseille on how computational modelling can be applied to cancer research.</p>
           <p style="text-align:justify;">•	2010-2014 <a href="https://talks.cam.ac.uk/show/archive/26320" target="_blank" rel="noopener">Science Society member</a>, Wolfson College Cambridge: Inviting public lecture speakers, organising their lectures and entertaining them during their visit.</p>
           <p style="text-align:justify;">•	2013-14 guest lecturer at UCL's <a href="https://www.ucl.ac.uk/wolfson-institute-biomedical-research/msc-drug-design" target="_blank" rel="noopener">MSc in Drug Design</a>, on computational methods for drug design.</p>
           <p style="text-align:justify;">•	2013 guest lecturer at Cambridge's <a href="https://www.ccbi.cam.ac.uk/" target="_blank" rel="noopener">Computational Biology Institute</a> on <a href="https://talks.cam.ac.uk\talk\index\43654" target="_blank" rel="noopener">ML 
            approaches to predicting protein-ligand binding</a>.</p>
           <p style="text-align:justify;">•	2011 Organised a 1-day visit to <a href="https://www.gsk.com/" target="_blank" rel="noopener">GSK</a> for EMBL-EBI and UCL postgraduates: lectures, networking and career opportunities in industry.</p> 
</div>
</div>
</div>
</div>
</section>    

  
<section id="news" class="home-section wg-experience">
  <div class="home-section-bg">
  </div>
  <div class="container">
  <div class="row">
  <div class="section-heading col-12 col-lg-4 mb-3 mb-lg-0 d-flex flex-column align-items-center align-items-lg-start">
  <h1 class="mb-0">News</h1>
  </div>
  <div class="col-12 col-lg-8">

    <div class="row experience">
      <div class="col-auto text-center flex-column d-none d-sm-flex">
      <div class="row h-50">
      <div class="col border-right">&nbsp;</div>
      <div class="col">&nbsp;</div>
      </div>
      <div class="m-2">
      <!--For latest news only, otherwise not using exp-fill-->
      <span class="badge badge-pill border exp-fill">&nbsp;</span>
      </div>
      <div class="row h-50">
      <div class="col border-right">&nbsp;</div>
      <div class="col">&nbsp;</div>
      </div>
      </div>
      <div class="col py-2">
      <div class="card">
      <div class="card-body">
      <div class="d-flex align-content-start">
        <div>
      <div class="section-subheading card-title exp-title text-muted my-0">Postdoc available</div>
      <div class="section-subheading card-title exp-company text-muted my-0"><a href="https://www.jobs.ac.uk/job/DAT978/research-associate" target="_blank" rel="noopener">Three-year postdoc in AI for structure-based Virtual Screening</a></div>
      <div class="text-muted exp-meta">Deadline: 24 July 2023</div>
      </div>
      </div>
        <div class="text-align:justify;font-size: 1.25em">
        The link contains all the required information to apply. Please note the deadline above.
        </div>
      </div>
      </div>
      </div>
      </div>

    <div class="row experience">
      <div class="col-auto text-center flex-column d-none d-sm-flex">
      <div class="row h-50">
      <div class="col border-right">&nbsp;</div>
      <div class="col">&nbsp;</div>
      </div>
      <div class="m-2">
      <!--For latest news only, otherwise not using exp-fill-->
      <span class="badge badge-pill border exp-fill">&nbsp;</span>
      </div>
      <div class="row h-50">
      <div class="col border-right">&nbsp;</div>
      <div class="col">&nbsp;</div>
      </div>
      </div>
      <div class="col py-2">
      <div class="card">
      <div class="card-body">
      <div class="d-flex align-content-start">
        <div>
      <div class="section-subheading card-title exp-title text-muted my-0">Paper published</div>
      <div class="section-subheading card-title exp-company text-muted my-0"><a href="https://www.mdpi.com/2218-273X/13/3/498" target="_blank" rel="noopener">On the Best Way to Cluster NCI-60 Molecules</a></div>
      <div class="text-muted exp-meta">March 2023</div>
      </div>
      </div>
        <div class="text-align:justify;font-size: 1.25em">
        For virtual screening of chemically diverse libraries, your predictive model should be trained, tuned and validated using different clusters of molecules.
          But not anything goes here, as clustering quality can vary strongly with the employed algorithm and outlier detection method.
        </div>
      </div>
      </div>
      </div>
      </div>

    <div class="row experience">
      <div class="col-auto text-center flex-column d-none d-sm-flex">
      <div class="row h-50">
      <div class="col border-right">&nbsp;</div>
      <div class="col">&nbsp;</div>
      </div>
      <div class="m-2">
      <!--For latest news only, otherwise not using exp-fill-->
      <span class="badge badge-pill border">&nbsp;</span>
      </div>
      <div class="row h-50">
      <div class="col border-right">&nbsp;</div>
      <div class="col">&nbsp;</div>
      </div>
      </div>
      <div class="col py-2">
      <div class="card">
      <div class="card-body">
      <div class="d-flex align-content-start">
        <div>
      <div class="section-subheading card-title exp-title text-muted my-0">Paper published</div>
      <div class="section-subheading card-title exp-company text-muted my-0"><a href="https://pubs.acs.org/doi/10.1021/acs.jcim.3c00218" target="_blank" rel="noopener">Beware of Simple Methods for Structure-Based Virtual Screening: The Critical Importance of Broader Comparisons</a></div>
      <div class="text-muted exp-meta">February 2023</div>
      </div>
      </div>
        <div class="text-align:justify;font-size: 1.25em">
        Here we show how virtual screening performance can still be overestimated when using simple methods or training-test data partitions designed to be unbiased.
        </div>
      </div>
      </div>
      </div>
      </div>
    
  
  <div class="row experience">
  <div class="col-auto text-center flex-column d-none d-sm-flex">
  <div class="row h-50">
  <div class="col border-right">&nbsp;</div>
  <div class="col">&nbsp;</div>
  </div>
  <div class="m-2">
  <span class="badge badge-pill border">&nbsp;</span>
  </div>
  <div class="row h-50">
  <div class="col border-right">&nbsp;</div>
  <div class="col">&nbsp;</div>
  </div>
  </div>
  <div class="col py-2">
  <div class="card">
  <div class="card-body">
  <div class="d-flex align-content-start"><div>
  <div class="section-subheading card-title exp-title text-muted my-0">Paper published</div>
  <div class="section-subheading card-title exp-company text-muted my-0"><a href="https://onlinelibrary.wiley.com/doi/full/10.1002/advs.202201501" target="_blank" rel="noopener">Interpretable Machine Learning Models to Predict the Resistance of Breast Cancer Patients to Doxorubicin from Their microRNA Profiles</a>
  </div>
  <div class="text-muted exp-meta">July 2022</div>
  </div>
  </div>
    <div class="text-align:justify;font-size: 1.25em">
  Large-scale machine learning analysis leading to the discovery of highly predictive, robust and even interpretable predictors of breast cancer patient response to doxorubicin.</div>
  </div>
  </div>
  </div>
  </div>

    

</div>
</div>
</div>
</section>
  

<section id="join" class="home-section wg-featured  "  >
  <div class="home-section-bg " >
    </div>
     <div class="container">
       <div class="row  ">
          <div class="section-heading col-12 col-lg-4 mb-3 mb-lg-0 d-flex flex-column align-items-center align-items-lg-start">
          <h1 class="mb-0">Join us</h1></div>
          <div class="col-12 col-lg-16">
            <div class="portrait-title"><p> </p>
           <p style="text-align:justify;">We are always interested in hearing from the best prospective group members in any area directly related to our research 
            interests (computational chemistry, bioinformatics, machine learning, etc.). Please email <a href="mailto: mailto:p.ballester@imperial.ac.uk">p.ballester@imperial.ac.uk</a> with your CV, grades from degrees, publications and the funding opportunity you would like to apply for. Note that I have to be contacted well ahead of the deadline, as we have to apply for departmental support as well. Some funding opportunities per career stage are provided below:</p>
           <p style="text-align:justify;"><b>Postdocs</b><br>
            •	<a href="https://royalcommission1851.org/fellowships/research-fellowships" target="_blank" rel="noopener">Royal Commission 1851</a><br>
            •	<a href="https://www.leverhulme.ac.uk/early-career-fellowships" target="_blank" rel="noopener">Leverhulme Trust</a><br>
            •	<a href="https://ec.europa.eu/research/mariecurieactions/actions/individual-fellowships_en" target="_blank" rel="noopener">Marie Sklodowska-Curie Individual Fellowship</a><br>
            •	<a href="https://www.imperial.ac.uk/admin-services/international-relations/bcstemeaf/" target="_blank" rel="noopener">British Council Women in STEM Fellowship (South Asia)</a><br>
            •	<a href="https://www.hfsp.org/funding/hfsp-funding/postdoctoral-fellowships" target="_blank" rel="noopener">HFSP Postdoctoral Fellowships</a></p>


            <p style="text-align:justify;"><b>PhD student</b><br>
            •	<a href="http://www.imperial.ac.uk/study/pg/fees-and-funding/scholarships/presidents-phd-scholarships/" target="_blank" rel="noopener">President's PhD Scholarship Scheme</a><br>
            •	<a href="https://www.imperial.ac.uk/study/fees-and-funding/postgraduate-taught/grants-scholarships/international-scholarship-collaborations/csc/" target="_blank" rel="noopener">China Scholarship Council (China)</a><br>
            •	<a href="https://www.imperial.ac.uk/study/fees-and-funding/postgraduate-taught/grants-scholarships/international-scholarship-collaborations/ims/" target="_blank" rel="noopener">Imperial Marshall Scholarship (USA)</a><br>
            •	<a href="https://www.imperial.ac.uk/study/fees-and-funding/postgraduate-taught/grants-scholarships/" target="_blank" rel="noopener">All Scholarship Schemes (including a scholarship search tool)</a><br>
            •	<a href="https://www.imperial.ac.uk/bioengineering/study/postgraduate-research/doctoral/funding/studentships/" target="_blank" rel="noopener">Bioengineering Department Studentships</a> <br>
            •	<a href="https://www.imperial.ac.uk/study/fees-and-funding/postgraduate-taught/grants-scholarships/international-scholarship-collaborations/conicyt-imperial-scholarship/" target="_blank" rel="noopener">Conicyt-Imperial Scholarship (Chile)</a><br>
            •	<a href="https://cscuk.fcdo.gov.uk/scholarships/high-income-commonwealth-phd-scholarships/" target="_blank" rel="noopener">High income Commonwealth PhD Scholarships (eligible countries within)</a><br>
            •	<a href="https://cscuk.fcdo.gov.uk/scholarships/commonwealth-phd-scholarships-for-least-developed-countries-and-fragile-states/" target="_blank" rel="noopener">Commonwealth PhD Scholarships (eligible countries within)</a><br>
            •	<a href="https://cscuk.fcdo.gov.uk/scholarships/commonwealth-split-site-scholarships-for-low-and-middle-income-countries/" target="_blank" rel="noopener">Commonwealth Split-site Scholarship (eligible countries within)</a></p>
            
            <p style="text-align:justify;">Information about the PhD programme in the Department of Bioengineering 
              can be found <a href='https://www.imperial.ac.uk/bioengineering/study/postgraduate-research/doctoral/' target="_blank" rel="noopener">here</a>. For general information on the tuition fees and cost of living in London, 
              please read this <a href='http://www.imperial.ac.uk/studentfinance' target="_blank" rel="noopener">link</a>. For other sources of PhD funding you can also have a look 
              <a href='https://www.imperial.ac.uk/students/fees-and-funding/postgraduate-funding/postgraduate-scholarships/' target="_blank" rel="noopener">here</a>, 
              <a href='https://www.imperial.ac.uk/bioengineering/study/postgraduate-research/doctoral/funding/' target="_blank" rel="noopener">here</a> (BioEngineering funding) and 
              <a href='http://www.imperial.ac.uk/study/ug/fees-and-funding/' target="_blank" rel="noopener">here</a> (fees and funding). Some external scholarships can also be found 
              <a href='https://www.imperial.ac.uk/study/fees-and-funding/other-funding-sources/external-scholarships/' target="_blank" rel="noopener">here</a>.
            Before contacting me, please check that you fulfill the <a href='https://www.imperial.ac.uk/bioengineering/study/postgraduate-research/doctoral/apply/' target="_blank" rel="noopener">pre-requisites for application and entry</a> set by Imperial College.</p> 
            
            <p style="text-align:justify;"><b>Undergraduate or master students </b><br>
            You only have to email me your CV, grades from degrees, and course details.</p>
            
</div>
</div>
</div>
</div>
</section>  
  

  
  <section id="contact" class="home-section wg-pages  "  >
   <div class="home-section-bg " >
     
   </div>
    <div class="container">

      <div class="row  ">
      
        
          <div class="section-heading col-12 col-lg-4 mb-3 mb-lg-0 d-flex flex-column align-items-center align-items-lg-start">
            <h1 class="mb-0">Contact</h1>
            
          </div>

<div class="col-12 col-lg-8">


<div class="pub-list-item view-citation" style="margin-bottom: 1rem">
  <p><b>Email:</b> <a href="mailto:p.ballester@imperial.ac.uk">p.ballester@imperial.ac.uk</a></p>
  <p><b>Address: </b>Sir Michael Uren Hub, Imperial College White City campus, 86 Wood Lane, London - W12 0BZ, UK   </p>
  <p><b>How to get here:</b> <a href="https://www.imperial.ac.uk/visit/campuses/white-city/" target="_blank" rel="noopener">https://www.imperial.ac.uk/visit/campuses/white-city/</a> </p>
  <p><b>About this campus:</b> <a href="https://www.imperial.ac.uk/white-city-campus/" target="_blank" rel="noopener">https://www.imperial.ac.uk/white-city-campus/</a> </p>
  <img width="750" height="750" src="members/imperialWC.jpg" alt="The Ballester Group">
</div>
    
      </div>
    

    </div>
  </section>



  </div>

  <div class="page-footer">
    
    
    <div class="container">
      <footer class="site-footer">
  
</footer>

    </div>
    
  </div>

  
<div id="modal" class="modal fade" role="dialog">
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <h5 class="modal-title">Cite</h5>
        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
          <span aria-hidden="true">&times;</span>
        </button>
      </div>
      <div class="modal-body">
        <pre><code class="tex hljs"></code></pre>
      </div>
      <div class="modal-footer">
        <a class="btn btn-outline-primary my-1 js-copy-cite" href="#" target="_blank">
          <i class="fas fa-copy"></i> Copy
        </a>
        <a class="btn btn-outline-primary my-1 js-download-cite" href="#" target="_blank">
          <i class="fas fa-download"></i> Download
        </a>
        <div id="modal-error"></div>
      </div>
    </div>
  </div>
</div>

      

    
    <script src="/js/vendor-bundle.min.b73dfaac3b6499dc997741748a7c3fe2.js"></script>

        
        <script src="https://cdn.jsdelivr.net/gh/desandro/imagesloaded@v4.1.4/imagesloaded.pkgd.min.js" integrity="sha512-S5PZ9GxJZO16tT9r3WJp/Safn31eu8uWrzglMahDT4dsmgqWonRY9grk3j+3tfuPr9WJNsfooOR7Gi7HL5W2jw==" crossorigin="anonymous"></script>
        <script src="https://cdn.jsdelivr.net/gh/metafizzy/isotope@v3.0.6/dist/isotope.pkgd.min.js" integrity="sha512-Zq2BOxyhvnRFXu0+WE6ojpZLOU2jdnqbrM1hmVdGzyeCa1DgM3X5Q4A/Is9xA1IkbUeDd7755dNNI/PzSf2Pew==" crossorigin="anonymous"></script>
      
        
        <script src="https://cdn.jsdelivr.net/gh/highlightjs/cdn-release@10.2.1/build/highlight.min.js" integrity="sha512-Ypjm0o7jOxAd4hpdoppSEN0TQOC19UtPAqD+4s5AlXmUvbmmS/YMxYqAqarQYyxTnB6/rqip9qcxlNB/3U9Wdg==" crossorigin="anonymous"></script>
        
        
        <script src="https://cdn.jsdelivr.net/gh/highlightjs/cdn-release@10.2.1/build/languages/r.min.js" crossorigin="anonymous"></script>
        
      
      
      <script id="search-hit-fuse-template" type="text/x-template">
        <div class="search-hit" id="summary-{{key}}">
          <div class="search-hit-content">
            <div class="search-hit-name">
              <a href="{{relpermalink}}">{{title}}</a>
              <div class="article-metadata search-hit-type">{{type}}</div>
              <p class="search-hit-description">{{snippet}}</p>
            </div>
          </div>
        </div>
      </script>
      
        <script src="https://cdn.jsdelivr.net/gh/krisk/Fuse@v3.2.1/dist/fuse.min.js" integrity="sha512-o38bmzBGX+hD3JHWUFCDA09btWaqrNmoJ3RXLlrysA7PP01Kgs4UlE4MhelE1v5dJR3+cxlR4qQlotsW7jKsnw==" crossorigin="anonymous"></script>
        <script src="https://cdn.jsdelivr.net/gh/julmot/mark.js@8.11.1/dist/jquery.mark.min.js" integrity="sha512-mhbv5DqBMgrWL+32MmsDOt/OAvqr/cHimk6B8y/bx/xS88MVkYGPiVv2ixKVrkywF2qHplNRUvFsAHUdxZ3Krg==" crossorigin="anonymous"></script>
      
  
    
    
    <script id="page-data" type="application/json">{"use_headroom":false}</script>

     
    
    <script src="/en/js/wowchemy.min.247fd8f54253895301106e3006f53f38.js"></script>

    

</body>
</html>
